tests/liveserver_fixtures.py

```diff
@@ -152,7 +153,23 @@ def post_auth():
 
     @app.route('/api/v1/<pid>/<lid>/predict', methods=["POST"])
     def predict_sinc(pid, lid):
+        body = flask.request.data
+        body = body.decode('utf-8').splitlines()
+        rows = len(body) - 1
+
+        try:
+            # for files with row_id as first column
+            first_row = int(body[1].split(',')[0]) % 10
```

Comment:
in body of request to this endpoint we have csv file with header and row_id in first column^ so this line just extracts starting_row_id of incoming dataset.

This endpoint is used in some other tests as well, and these places do not follow this pattern, so for them we return just complete static entry as before